audit/ask-cal/source/constants/categories.ts

```diff
@@ -0,0 +1,14 @@
+export type CodeExampleCategory =  
+  | "Syntax example"
+  | "Usage example"
+  | "Non-MongoDB command"
+  | "Example return object"
+  | "Example configuration object";
+  
+export const CodeExampleDisplayValues: Record<CodeExampleCategory, string> = {
```

Comment:
Typically, I'd use an enum for this kind of thing. For example:
``` ts
enum CodeExampleDisplayValues {
  SyntaxExample: "Syntax example",
  UsageExample: "Usage example",
  ...
}
```

This StackOverflow has a good answer about `const` vs `enum` in TypeScript if you're curious: https://stackoverflow.com/questions/66862421/enum-vs-as-const